libraries/azure-app-configuration-importer/src/featureFlag.ts

```diff
@@ -0,0 +1,127 @@
+import { FeatureFlagValue as FeatureFlagValueV1 } from "@azure/app-configuration";
+import { ClientFilter } from "./models";
+
+export interface FeatureFlagValue extends FeatureFlagValueV1 {
+    id: string;
```

Comment:
Since `FeatureFlagValue`  (from the library) already has "id, enabled, displayName", why do we define them again ? I was thinking it would something like this 
```
export interface FeatureFlagValue extends FeatureFlagValueV1 { 
    allocation?: Allocation;
    variants?: Variant[];
    telemetry?: {
        enabled?: boolean;
    };

}
```

Reply:
It looks like there is conflict between the FeatureFlagValue defined in the sdk and the new schema. In the sdk the filters are defined in PascalCase https://github.com/Azure/azure-sdk-for-js/blob/1be4937a08a0d5f55a0a9621ed3670aecd670e93/sdk/appconfiguration/app-configuration/src/featureFlag.ts#L34 however in the new schema everything is in underscore case. I have removed extending the interface from the sdk as it is causing conflicts.

Reply:
The field names in this type don't have to be consistent with the name in json schema. They don't need to have 1:1 map.

Reply:
@ChristineWanjau  I would suggest to extend the `FeatureFlagValue` type in SDK